views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,10 @@
+<img src='{{image_url}}' alt='{{name}}'>
+
+<a href="/beers/beer-i3f4d34s34b"><img src="{{image_url}}" alt=""></a>
+
```

Comment:
you have the image twice here :)

Reply:
Also, I will assume you have just started with a hard coded url id and then didn't have enough time to finish. Let me know if you have any questions about this.